src/microbe_stage/Microbe.Interior.cs

```diff
@@ -60,6 +68,12 @@ public partial class Microbe
 
     private float timeUntilChemoreceptionUpdate = Constants.CHEMORECEPTOR_COMPOUND_UPDATE_INTERVAL;
 
+    /// <summary>
+    ///   The stored compounds in this microbe
+    /// </summary>
+    [JsonProperty]
+    public CompoundBag Compounds { get; private set; } = new(0.0f);
```

Comment:
Why is this no longer readonly?

Reply:
It's a property?? It implements `IEngulfable.Compounds`.

Reply:
Oh yeah, I didn't realize that change makes it not possible for this to be readonly anymore. I guess it's good enough, though this is now less protected against accidentally someone reassigning this object in this class.

Reply:
I'd feel better if there was a comment saying to never reassign this or if this was a property with a readonly backing field (and the json attribute was on the backing field to make save/load work).

Reply:
Added backing readonly field.